geometry_msgs/Point[] waypoints
geometry_msgs/Point[] x_train
int64 sampling_rate
bool use_altitude